plugins/parsers/dropwizard/parser.go

```diff
@@ -227,11 +227,7 @@ func (p *parser) readDWMetrics(metricType string, dwms interface{}, metrics []te
 			parsed, err := p.seriesParser.Parse([]byte(measurementName))
```

Comment:
I'm wondering if this error needs to be returned instead?

Reply:
It looks like it's handled in the if statement two lines below, I'm unsure if that's what you meant though

Reply:
Yeah mainly just thought why are we nil checking it but then not either returning the error or logging it. I'm aware this isn't related to what you've changed so probably fine to leave it as is.

Reply:
I agree, although as it's out of scope for this PR I'll probably leave it. and thanks for reviewing!

Reply:
yeah this logic is definitely bad. there's a panic hiding in here too. You don't have to fix it in this PR, but please circle back to it and/or open an issue